commitizen/commands/commit.py

```diff
@@ -61,7 +62,16 @@ def prompt_commit_questions(self) -> str:
 
         if not answers:
             raise NoAnswersError()
-        return cz.message(answers)
+
+        message = cz.message(answers)
+        message_len = len(message.partition("\n")[0])
```

Comment:
I didn't know we could use `partition` 👀 Great job

Reply:
should we `strip()` as well?

Reply:
Sure, thanks for the suggestion!